soak/prow/README.md

```diff
@@ -0,0 +1,116 @@
+## Introduction
+This document serves as the onboarding guide for AWS service teams to automate the soak testing for service controller using
+[ACK prow cluster](https://prow.ack.aws.dev/) .
+
+### Infrastructure Components
+1. Prow cluster owned by ACK team, where the postsubmit job will execute.
+2. An EKS cluster owned by service team. 
+3. Monitoring resources in the EKS cluster to monitor soak test runs. Ex: Prometheus and Grafana. 
+4. The IAM role which is used to create the EKS cluster in #2. This role will have trust relationship for ACK service so
+   that ACK prow cluster can communicate the EKS cluster executing soak tests. 
+4. The IAM role for service accounts to provide access to AWS for service controller and soak runner.
+
+### Steps Overview
+1. Whenever service teams cut a new semver release(Ex: v0.0.2) for the service controller, a postsubmit prow job is
+triggerred.
+2. The postsubmit prowjob assumes the IAM role used by service teams to create the soak test EKS cluster.
+3. The postsubmit prowjob uses EKS `update-kubeconfig` command to gain access to the soak test cluster.
+4. Based on the artifacts in the latest release, prowjob starts the service controller in the soak test cluster.
+5. Prowjob will build a container image which will execute the soak tests
+   against the controller installed in #4 .
+6. Prowjob creates a Kubernetes Job resource in Soak test cluster and waits for the job to complete.
+7. Once the job completes, Prowjob cleans up the Job resource from #6 and the service controller resources from #4 .
+8. Based on the monitoring and alarming setup by service teams in the soak test cluster, they can review the results of
+   soak tests.
+
+## Onboarding Guide
+
+### Prerequisite
+* An AWS account to host the cluster. This will be reused from service teams onboarding for e2e test executions.
+* [yq](https://mikefarah.gitbook.io/yq/#install)
+* [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
+* [helm](https://helm.sh/docs/intro/install/)
+
+### Steps
+1. Update the current shell with AWS Credentials of ack-test role. Replace "<my-service-name>" with actual service name
```

Comment:
Would you mind going back to the general onboarding document and clarifying that the `ack-test-role` will need EKS admin privileges (to create the cluster)